Craigslist/Craigslist/Views/Listing/Publish.cshtml

```diff
@@ -4,21 +4,51 @@
     <div id="publish-ad-validation-summary">
         @Html.ValidationSummary()
     </div>
-    <div id="publish-ad-contact-datails">
-        <label>Your contact information:</label>
-        @Html.TextBoxFor(i => i.Listing.Contact.FirstName, new { placeholder = "First Name" })
-        @Html.TextBoxFor(i => i.Listing.Contact.LastName, new { placeholder = "Last Name" })
-        @Html.TextBoxFor(i => i.Listing.Contact.Email, new { placeholder = "Email", type = "email" })
-        @Html.TextBoxFor(i => i.Listing.Contact.Phone, new { placeholder = "Phone" })
-    </div>
-    <div id="publish-ad-topbar">
-        <label>Title:</label>
-        @Html.TextBoxFor(i => i.Listing.Header, new { id = "publish-ad-header" })
-        <label>Price:</label>
-        @Html.TextBoxFor(i => i.Listing.Price, new { type = "number" })
-        <label>Category:</label>
-        @Html.DropDownListFor(i => i.Listing.CategoryId, @Model.Categories)
-    </div>
-    @Html.TextAreaFor(i => i.Listing.Body, new { id = "publish-ad-body" })
-    <button type="submit">Publish</button>
+    <div id="publish-ad-contact-datails">

```

Comment:
Content - to align to the top and make the jumbotron height adjusted to the actual content height.

icons: to add margin to the right (5 px)